.github/workflows/package-browser-sdk.yml

```diff
@@ -8,24 +8,26 @@ permissions:
 
 env:
   NPM_API_KEY: ${{ secrets.NPM_API_KEY }}
+  GO_VERSION: "1.22"
+  DAGGER_VERSION: "0.12.3"
```

Comment:
dagger recently removed a bunch of things: https://github.com/dagger/dagger/releases/tag/v0.13.0

going to pin the version to what we have in `flipt`, maybe we could consider bumping everything to the latest `0.12.x` version though